packages/inter-protocol/src/proposals/addAssetToVault.js

```diff
@@ -18,7 +18,8 @@ export * from './startPSM.js';
  * @property {number} [decimalPlaces]
  * @property {string} [proposedName]
  * @property {string} keyword
- * @property {string} oracleBrand
+ * @property {string} [issuerName]
+ * @property {string} [oracleBrand]
```

Comment:
The new changes since last testing period look good to me (note: just from reading the code, i have yet to try a deployment).  I might suggest adding these comments, or something along these lines, to add more clarity around the parameters:

```suggestion
 * @property {string} keyword - the issuer keyword record for the contract instance
 * @property {string} [issuerName] - defaults to `keyword` if not provided
 * @property {string} [proposedName] - defaults to `issuerName` if not provided
 * @property {string} [oracleBrand] - defaults to `issuerName` if not provided
 ```